scripts/output/projects/FSDP_collect.R

```diff
@@ -236,9 +246,11 @@ val <- as.data.table(read.quitte(val))
 saveRDS(val, file = file.path("output", paste(rev, "FSDP_validation.rds", sep = "_")), version = 2, compress = "xz")
 
 message("Plotting figures ...")
-heatmapFSDP(reg, tableType = 1,    file = file.path("output", paste(rev, "FSDP_heatmap1.jpg", sep = "")))
-heatmapFSDP(reg, tableType = "2a", file = file.path("output", paste(rev, "FSDP_heatmap2a.jpg", sep = "")))
-heatmapFSDP(reg, tableType = 3,    file = file.path("output", paste(rev, "FSDP_heatmap3.jpg", sep = "_")))
-spatialMapsFSDP(reg, iso, grid, reg2iso, file = file.path("output", paste(rev, "FSDP_spatialMaps.jpg", sep = "_")))
-supplPlotsFSDP(reg, scenarioType = "all", file = file.path("output", paste(rev, "FSDP_supplPlots.jpg", sep = "_")))
-SupplPlotsCropShr(gdx = gdx, file = file.path("output", paste(rev, "FSDP_supplPlotCropShr.jpg", sep = "_")) )
\ No newline at end of file
+heatmapFSDP(reg, tableType = 1,    file = file.path("output", paste(rev, "FSDP_heatmap1.png", sep = "_")))
```

Comment:
```suggestion
heatmapFSDP(reg, tableType = 1,    file = file.path("output", paste0(rev, "_FSDP_heatmap1.png")))
```
Just a tiny bit shorter, and maybe slightly easier to read, because there's one argument less. Take it or leave it :wink: